### PR TITLE
[script.service.janitor] 4.4.0

### PR DESCRIPTION
**Pull request by Anthirian**
### Description
- [NEW]  Leia compatibility
- [NEW] Updated log viewer to use Estuary skin
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Anthirian** [comment]
So is there still something preventing the pull request from being merged? I see no activity here 😅

File: script.service.janitor/addon.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+[B]Attention![/B]
+Cet addon à la faculté de supprimer entièrement votre collection vidéo! Assurez vous de bien comprendre à quoi est destiné cet addon avant d'activer un nettoyage dans les paramètres. Je ne pourrait aucunement être tenu responsable en cas de perte de fichier lié à une mauvaise utilisation.
+
+[I]developé par Andrew 'drewzh' Higginson (xbmcfilecleaner@drewzh.com)[/I]
+        </description>
+        <disclaimer lang="en_GB">It is recommended to not delete videos permanently until you are familiar with all of the settings.</disclaimer>
+        <disclaimer lang="nl_NL">Het is verstandig om videos pas permanent te verwijderen als je bekend bent met alle instellingen.</disclaimer>
+        <disclaimer lang="fr_FR">Il est recommandé de ne pas supprimer physiquement de vidéos avant de bien maitriser les réglages.</disclaimer>
+        <platform>all</platform>
+        <language>en nl fr</language>
```
Comment (Rechi) [inline script.service.janitor/addon.xml]: The language is for indication the language of the content the add-on provides. As this add-on doesn't provide content on it's own either remove the tag or set it to an empty value.
```suggestion
```

File: script.service.janitor/viewer.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    It is used to display the contents of a log file, and as such uses a fullscreen window to show as much text as
+    possible. It also contains two buttons for trimming and clearing the contents of the log file.
+    """
+    TEXTBOXID = 202
+    TRIMBUTTONID = 301
+    CLEARBUTTONID = 302
+
+    def __init__(self, xml_filename, script_path, default_skin="Default", default_res="720p", *args, **kwargs):
+        self.log = utils.Log()
+        WindowXMLDialog.__init__(self)
```
Comment (Rechi) [inline script.service.janitor/viewer.py]: [Documentation](https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__window__dialog__xml.html)  tells that `xmlFilename` and `scriptPath` are required arguments for this function. Does it work correctly without passing those arguments?
```suggestion
        WindowXMLDialog.__init__(self, xml_filename, script_path)
```

Comment (Anthirian) [reply script.service.janitor/viewer.py]: It's always worked for me this way, even when developing the new log file viewer based on the Estuary skin the window opened just fine everytime.

**Comment by Rechi** [comment]
> Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Please squash the commits.

**Comment by Anthirian** [comment]
> > Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0
> 
> Please squash the commits.

Okay, done.

**Comment by Anthirian** [comment]
Any other issues I should fix?

**Comment by Rechi** [comment]
@Anthirian this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Pull request merged**